### DISCUSSION TITLE
implementing rounded borders

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#56] [Q&A] [answered]
How does CSS3 help in implementing rounded borders easily?

**Comment by Bunny77K9** [comment, marked as answer]
CSS3 has the <border-radius> property that allows elements to be created with nice-looking rounded corners. This can easily be applied to all four sides or as per requirement.

The <border-radius> property has four attributes for four corners:

    <border-top-left-radius>
    <border-top-right-radius>
    <border-bottom-left-radius>
    <border-bottom-right-radius>
